webapp/src/core.ts

```diff
@@ -485,6 +485,14 @@ export function scrollIntoView(item: JQuery, margin = 0) {
     }
 }
 
+export function resetFocus() {
+    let accessibleMenu = document.getElementById('accessibleMenu');
+    accessibleMenu.tabIndex = 0;
+    accessibleMenu.focus();
+    accessibleMenu.blur();
+    accessibleMenu.tabIndex = -1;
```

Comment:
This doesn't look right? why are we reseting the focus on the accessibleMenu? why not the document / root?

Reply:
So it's weird but resetting it to the body didn't worked for me on Edge. It was still focusing the simulator.
Also, just blurring the element or giving the focus to an element that doesn't had a tabindex was not working on Edge and Chrome.

Reply:
I can do it on the element with the 'content' id if you prefer.